storage/replica_trigger.go

```diff
@@ -320,26 +320,38 @@ func (r *Replica) leasePostCommitTrigger(
 					r.gossipFirstRangeLocked(ctx)
 				}()
 			}
-		} else if trigger.lease.Covers(r.store.Clock().Now()) {
-			if err := r.withRaftGroup(func(raftGroup *raft.RawNode) error {
-				if raftGroup.Status().RaftState == raft.StateLeader {
-					// If this replica is the raft leader but it is not the new lease
-					// holder, then try to transfer the raft leadership to match the
-					// lease.
-					log.Infof(ctx, "range %v: replicaID %v transfer raft leadership to replicaID %v",
-						r.RangeID, replicaID, trigger.lease.Replica.ReplicaID)
```

Comment:
I removed the `TODO(tschottdorf)` I added in this PR. Not sure if you're referring to another one. Your comments don't seem to be anchored to the code correctly.